services/api/tests/test_code_reviewer_persona.py

```
@@ -359,8 +359,10 @@ def test_reviewed_response_has_no_degraded_reason() -> None:
 
 def test_diff_hunk_rejects_missing_at_at_in_body() -> None:
     """Boundary check — catches a parser regression that loses the @@
-    header before the body string is captured."""
+    header before the body string is captured. Raises DiffParseError
+    (the exception the dispatch degrade contract catches), never a bare
+    AssertionError that would escape it and poison the consumer."""
     import pytest as _pytest
```

Ruff (0.15.20)

[warning] Incorrect import of `pytest`; use `import pytest` instead

(PT013)